core/packages/contracts/src/BeefyClient.sol

```diff
@@ -463,6 +463,8 @@ contract BeefyClient is Ownable {
     function encodeCommitment(Commitment calldata commitment) internal pure returns (bytes memory) {
         return bytes.concat(
             commitment.payload.prefix,
+            hex"6d68", // "mh" (MMR_ROOT_ID)
+            hex"80", // 128 (32 << 2, compact encoding)
```

Comment:
I think a more meaningful comment for auditors would be to say that this compact u64 represents the length of the mmrRootHash.

Reply:
Tried out actually encoding `"mh"` and `32`, left the id's encoding in and hard-coded the length.